_examples/finally/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+// Example application using the github.com/briandowns/spinner API
+
+import (
+	"time"
+
+	"github.com/briandowns/spinner"
+)
+
+func main() {
+	s := spinner.New(spinner.CharSets[9], 100*time.Millisecond) // Build our new spinner
+	s.Color("red")                                              // Set the spinner color to red
+	s.Finally = "Complete!\nNew line!\nAnother one!\n"
```

Comment:
Should this be: 
```
s.FinalMSG = "Complete!\nNew line!\nAnother one!\n"
```
Finally doesn't seem to be In the [structure](https://github.com/briandowns/spinner/pull/27/files#diff-b96c3329356c73ae67250a025d023a90R96)